www/_posts/2015-11-11-cordova-windows-4.2.0.md

```diff
@@ -0,0 +1,44 @@
+---
+layout: post
+author:
+    name: Sergey Grebnov
+    url: https://twitter.com/sgrebnov
+title:  "Apache Cordova Windows 4.2.0"
+categories: announcements
+tags: news releases
+---
+
+We are happy to announce that `Cordova Windows 4.2.0` has been released!
+
+This release adds support for **back button** handling on **Windows 10** and
+**Windows Phone 8.1** and various other improvements. It will be the default
+Windows version after next `cordova-cli` release.
```

Comment:
Grammar: It will be the default windows version after *the* next `cordova-cli` release.